utPlayer.py

```diff
@@ -52,20 +52,21 @@ def __init__(self, session:object, playerTool:dict={'tool': 'gstreamer', 'prereq
         for cmd in playerTool["prerequisites"]:
             self.session.write(cmd)
 
-    def play(self, streamFile:str):
+    def play(self, streamFile:str, arguments:str = ""):
```

Comment:
what sort of args do you need to pass?

I was thinking is that based on platform the arguments are internal to this module, rather than being passed in, similar to the way that `dsClass` works, it's completely self contained.

so we need to configure this module based on configuration requirement which is all internal to the class.

Reply:
As discussed moved player configuration to the class